apps/backend/src/modules/activity-logs/activity-logs.ts

```diff
@@ -2,18 +2,28 @@ import { logger } from "~/libs/modules/logger/logger.js";
 import { contributorService } from "~/modules/contributors/contributors.js";
 import { gitEmailService } from "~/modules/git-emails/git-emails.js";
 import { projectApiKeyService } from "~/modules/project-api-keys/project-api-keys.js";
+import { ProjectModel } from "~/modules/projects/project.model.js";
+import { ProjectRepository } from "~/modules/projects/project.repository.js";
+import { ProjectService } from "~/modules/projects/project.service.js";
 
 import { ActivityLogController } from "./activity-log.controller.js";
 import { ActivityLogModel } from "./activity-log.model.js";
 import { ActivityLogRepository } from "./activity-log.repository.js";
 import { ActivityLogService } from "./activity-log.service.js";
 
+const projectRepository = new ProjectRepository(ProjectModel);
+const projectService = new ProjectService(
+	projectRepository,
+	projectApiKeyService,
+);
+
```

Comment:
Let's import `projectService` as other services instead of creating new one. The fact it (or repo only) was created inside `project-api-keys.ts` is that `project-api-keys.ts` was imported inside `projects.ts` in order to inject `projectApiKeyService` into `ProjectService`, so there was an issue with cyclic dependencies.

Reply:
Oh, thanks for noticing